source/frontend/src/features/properties/map/project/detail/ProjectProductView.tsx

```diff
@@ -0,0 +1,58 @@
+import { Section } from 'features/mapSideBar/tabs/Section';
+import { SectionField } from 'features/mapSideBar/tabs/SectionField';
+import { Api_Product, Api_Project } from 'models/api/Project';
+import styled from 'styled-components';
+import { formatMoney, prettyFormatDate } from 'utils';
+
+export interface IProjectProductViewProps {
+  project?: Api_Project;
+}
+
+const ProjectProductView: React.FunctionComponent<
+  React.PropsWithChildren<IProjectProductViewProps>
+> = ({ project }) => {
+  const productCount = project?.products?.length || 0;
+
+  return (
+    <StyledSummarySection>
+      <Section header="Associated Products" isCollapsable initiallyExpanded>
+        {project?.products?.map((a: Api_Product, b: number) => (
```

Comment:
could we rename a to something like product or currentProduct and b to index or similar?